internal/server/apimigrator.go

```diff
@@ -102,7 +102,7 @@ func rebuildRequest(c *gin.Context, newReqObj interface{}) {
 	for i := 0; i < r.NumField(); i++ {
 		f := r.Field(i)
 		if fieldName, ok := t.Field(i).Tag.Lookup("form"); ok {
-			if structNameWithPkg(f.Type()) == "uid.ID" {
+			if f.Type() == reflect.TypeOf(uid.ID(0)) {
```

Comment:
I also removed `structNameWithPkg` and replaced it with these more strict type checks.  I think the type comparison is nicer because an IDE can find these references, but it's not going to find `uid.ID`.

This `apimigrator` was the last place that used it after I removed it from `openapi.go`.